src/integrations/uport/index.js

```diff
@@ -0,0 +1,116 @@
+import { getTokenBalance } from 'api'
+import { setConnectionStatus } from 'actions/blockchain'
+import { WALLET_PROVIDER } from 'integrations/constants'
+import InjectedWeb3 from 'integrations/injectedWeb3'
+import { fetchTournamentUserData } from 'routes/Scoreboard/store/actions'
+import { weiToEth, hexWithoutPrefix } from 'utils/helpers'
+import { getTokenAddress } from 'utils/configuration'
+import initUportConnector, { connect, connectorLogOut, isUserConnected } from './connector'
+
+export const notificationsEnabled = false
```

Comment:
Use config here? Maybe it would make sense to turn the `provider` key into an object, supplying a `defaultProvider` and `options` that can be used in the actual integrations (and would be specific to them)